hirola/front/static/front/css/landing_page.css

```diff
@@ -1,2 +1,59 @@
-html, body {height: 100%; margin:0; padding:0;     flex: 1;
-  overflow-y: scroll;}
\ No newline at end of file
+html,
+body {
+    /* height: 100%; */
+    /* margin: 0;
+    padding: 0; */
+    /* flex: 1; */
+    /* overflow-y: scroll; */
+    /* height: 100vh; */
+}
+
+.carousel {
+    height: auto;
+    height: 93.5vh;
+}
+.content {
+    margin-bottom: 0 !important;
+}
+
+.item-one {
+    position: relative;
+    background-image: url('../../../static/front/phone8.jpeg');
```

Comment:
Ideally these image will be loaded from the database.  It will not be loaded statically as in this case where the url is hard coded. The reason for doing this is to help the one administering the website to easily change the images in future. This also is the same for the wordings. Also when it comes to html the administrator may want to have  5 and not 3 images that change. In flask and django this will be easily implemented if the url is on the html tag, by using a for loop to put all images on the html page. In this case adding python code on css will be difficult. Determine a manner in which the url for the image can be loaded from the html for item-one, two and three

Reply:
I agree, let me make the changes